### PR TITLE
fix: vector3 error is gone

### PR DESCRIPTION
**Pull request by Rinder5**

### CODE REVIEW

File: Assets/Scripts/PlayerController.cs [hunk 1]
```diff
@@ -35,12 +35,12 @@ public void UpdateCamera()
     // Update is called once per frame
     private void Update()
     {
-        if (Input.GetAxisRaw("Horizontal") != 0 || Input.GetAxisRaw("Vertical") != 0)
+        if (Input.GetAxis("Horizontal") != 0 || Input.GetAxis("Vertical") != 0)
         {
```
Comment (nicholaschuayunzhi) [inline Assets/Scripts/PlayerController.cs:39]: Was this change intended?

File: Assets/Scripts/PlayerController.cs [hunk 2]
```diff
@@ -35,12 +35,12 @@ public void UpdateCamera()
...
             Move();
         }
 
-        if (_onGround && Input.GetAxisRaw("Jump") == 1.0f)
+        if (_onGround && Input.GetAxis("Jump") == 1.0f)
         {
```
Comment (nicholaschuayunzhi) [inline Assets/Scripts/PlayerController.cs:44]: Same here

File: Assets/Scripts/PlayerController.cs (hunk 1 above)
Comment (Rinder5) [reply Assets/Scripts/PlayerController.cs:39]: yea, it makes the turning more natural. I just put in a clause to make sure that the facing doesn't change if vector is zero, like what you mentioned previously.

File: Assets/Scripts/PlayerController.cs (hunk 2 above)
Comment (Rinder5) [reply Assets/Scripts/PlayerController.cs:44]: same here

File: Assets/Scripts/PlayerController.cs (hunk 1 above)
Comment (nicholaschuayunzhi) [reply Assets/Scripts/PlayerController.cs:39]: Okays! Just make sure that the character faces the direction he is moving towards. Perhaps we can add something like a "nose" for testing purposes.

Comment (Rinder5) [reply Assets/Scripts/PlayerController.cs:39]: since we don't update only on the last frame, it should be facing the same direction as the last movement. If he is moving, then the facing is definitely on
